package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "maplibre",
+  "version": "1.0.0",
+  "description": "our website",
+  "main": "index.js",
+  "dependencies": {
+    "@fontsource/alata": "^4.5.7",
+    "bootstrap": "^5.1.3",
+    "maplibre-gl": "^2.1.9"
+  },
+  "scripts": {
+    "build": "hugo --minify",
```

Comment:
Hugo is not an npm package, right?

Reply:
No, but hugo is available in the official Ubuntu repositories. Not sure about windows though.

Reply:
Can you add a github workflow to build this website on push to main?

Reply:
done